spec/support/request/split_checkout_request_helper.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module SplitCheckoutRequestsHelper
```

Comment:
This isn't a request helper and probably shouldn't live in `/support/request/`; it's methods are checking things in the DOM (which doesn't happen in request specs) and it's being used for system tests.

Reply:
Ah I see the previous checkout helper is named like that. Well... it's wrong.

**Naming things correctly** in software development is both notoriously hard and deceptively important :sweat_smile:

Reply:
Great catch, thank you :+1: